assets/arc-0031/client/utils/hooks/useAuth.ts

```diff
@@ -1,17 +1,19 @@
-import { encode, decode } from '@msgpack/msgpack'
```

Comment:
If you run `npm run lint` you'll notice that the linter isn't happy :/

Reply:
It should be fixed now (just a couple of warnings)